__tests__/ratingRoutes.test.js

```diff
@@ -57,4 +83,101 @@ describe('Check ratings', () => {
         done();
       });
   });
+
+  describe('Create Rating', () => {
+    it('should throw a 500 when an error occurs on the server', (done) => {
+      const stub = sinon.stub(Rating, 'create').rejects(new Error('Foreign Key constraint'));
+      chai
+        .request(app)
+        .post(rateUrl)
+        .set('Authorization', `Bearer ${verifiedUserToken}`)
+        .send(mockRates[0])
+        .end((err, res) => {
+          expect(res.status).to.eql(500);
+          stub.restore();
+          done();
+        });
+    });
+
+    it('should create a rating successfully', (done) => {
+      chai
+        .request(app)
+        .post(`${rateUrl}`)
+        .set('Authorization', `Bearer ${verifiedUserToken}`)
+        .send(mockRates[0])
+        .end((err, res) => {
+          expect(res.status).to.equal(201);
+          const { article } = res.body;
+          expect(article.message).to.be.eql('Your rating has been added successfully');
+          expect(article).to.have.property('articleId');
+          expect(article).to.have.property('rate');
+          expect(article).to.have.property('firstName');
+          expect(article).to.have.property('lastName');
+          done();
+        });
+    });
+    it('should throw error when author tries to rate his article', (done) => {
+      chai
+        .request(app)
+        .post(authorUrl)
+        .set('Authorization', `Bearer ${verifiedUserToken}`)
+        .send(mockRates[1])
+        .end((err, res) => {
+          expect(res.status).to.equal(400);
+          expect(res.body.errors.message).to.be.eql("You can't rate your article");
+          done();
+        });
+    });
+    it('should update rating when author tries to rate an article again', (done) => {
+      chai
```

Comment:
Should this be for an author rating an article, or a user?

Reply:
Okay, I'll change that